ARCL/Source/Nodes/LocationAnnotationNode.swift

```diff
@@ -75,10 +68,15 @@ open class LocationAnnotationNode: LocationNode {
         if scaleRelativeToDistance {
             scale = appliedScale.y
             annotationNode.scale = appliedScale
+            childNodes.forEach { child in
+                child.scale = appliedScale
+            }
         } else {
-            //Scale it to be an appropriate size so that it can be seen
+            // Scale it to be an appropriate size so that it can be seen
             scale = Float(adjustedDistance) * 0.181
-            if distance > 3_000 { scale *=  0.75 }
+            if distance > 3_000 {
+                scale *= 0.75
+            }
 
             annotationNode.scale = SCNVector3(x: scale, y: scale, z: scale)
```

Comment:
I believe this section also need to iterate through the node's children and adjust their scale in accordance. Something like this should suffice

```
for childnode in locationNode.childNodes {
    childnode.scale = SCNVector3(x: scale, y: scale, z: scale)
}
```